webpack.config.babel.js

```diff
@@ -8,7 +8,7 @@ const env = process.env.NODE_ENV || 'development'
  * Base Configuration.
  */
 const config = {
-  devtool: 'cheap-eval-source-map',
+  devtool: env === 'production' ? '#eval' : '#eval-source-map',
```

Comment:
`devtool: "eval-source-map"` is really as good as `devtool: "source-map"`, but can cache SourceMaps for modules. It’s much faster for rebuilds.

`devtool: "eval"` has the best performance, but it only maps to compiled source code per module. In many cases this is good enough.

"#" is defaulted in WP2, but I like to add it for good measure